website/docs/r/workspace.html.markdown

```diff
@@ -70,9 +70,11 @@ The following arguments are supported:
 * `operations` - **Deprecated** Whether to use remote execution mode. When set to `false`, the workspace will 
   be used for state storage only. Defaults to `true`. This value _must not_ be provided if `execution_mode` is 
   provided.
-* `queue_all_runs` - (Optional) Whether all runs should be queued. When set
-  to `false`, runs triggered by a VCS change will not be queued until at least
-  one run is manually queued. Defaults to `true`.
+* `queue_all_runs` - (Optional) Whether all runs should be queued. When set to `false`, runs triggered by a webhook
+  (such as a commit in VCS) will not be queued until at least one run is manually queued. Although the Terraform Cloud API
+  defaults to `false` as a feature (allowing other settings to be changed before accepting webhooks), this provider
+  defaults the value to `true` as any workspace provisioned with `false` would need to then have a run manually queued out-of-band
+  before accepting webhooks.
```

Comment:
```suggestion
* `queue_all_runs` - (Optional) Whether the workspace should start automatically performing
  runs immediately after its creation. When set to `false`, runs triggered by a webhook
  (such as a commit in VCS) will not be queued until at least one run has been manually
  queued. Defaults to `true`. **Note:** In the Terraform Cloud API, this defaults to `false`
  instead, since setting variables or other settings might require additional imperative steps.
```